source_code/CDOCKER/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/inc)
+
+SET(SRC_FILES
+    ${CMAKE_CURRENT_LIST_DIR}/src/docker.c
+)
+
+add_library( cdockerinterface SHARED ${SRC_FILES})
```

Comment:
See comment above re naming. I would prefer `docker` or, if it's too confusing, `cdocker`

Reply:
Ok. I'll modify the naming.